zlp-scheduler/features/admin_view_schedule_submission.feature

```diff
@@ -0,0 +1,30 @@
+Feature: As an administrator, I want to see which students have submitted their schedules.
+  
+Background: Create Term and Cohort
+ Given The "New Test Term" terms exist 
```

Comment:
So, I have a step definition in `terms_steps.rb` that's "Given the following terms exist." Have you run all the cucumber scenarios to make sure there's not a regex matching problem with this step?

Reply:
Yes, all cucumber test will past.
There is no matching problem.

Reply:
Great!